lib/interactive.dart

```diff
@@ -126,6 +133,35 @@ class Interactive {
     return color;
   }
 
+  String _lookupFilterLabels(String filter, String option) {
+    String label = "Unknown";
+    for (var f in _filters) {
+      if (f.value == filter) {
```

Comment:
Consider inverting this to 

if (f.value != filter) continue;

It'll decrease the overall nesting level that makes the logic hard to read

Reply:
fixed